config/hooks/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd data-service
+npm run pretty-quick --staged && npm run lint
```

Comment:
This will run the analysis every time there is a commit to the repo, even java code changes.
Can we somehow restrict it to just run when there are changes in **data-service**?
if not, it's fine.